tableauserverclient/server/pager.py

```diff
@@ -16,6 +17,11 @@ def __init__(self, endpoint, request_opts=None):
             self._count = ((self._options.pagenumber - 1) * self._options.pagesize)
         else:
             self._count = 0
+            self._options = RequestOptions()
+
+        # Pager assumes deterministic order but solr doesn't guarantee sort order unless specified
+        if len(self._options.sort) == 0:
```

Comment:
Can just be `if not self._options.sort` -- any non empty set will be `True` and any empty set will be `False`